cli/tests/integrations/common.py

```diff
@@ -151,11 +151,11 @@ def wait_for_service(service_name, number_of_services=1, max_count=300):
         count += 1
 
 
-def add_job(app_path):
+def add_job(job_path):
```

Comment:
the issue with the build is app_path was changed to "job_path" on line 154, however line 164 also needs to change to job_path.   passes syntax check after the change.